.gitignore

```diff
@@ -0,0 +1,17 @@
+node_modules/
+
+# Installed npm modules
+package-lock.json
+node_modules
+
+# Folder view configuration files
```

Comment:
adding lots of stuff to the gitignore can become a giant bikeshed... we may want to avoid adding anything but node_modules for right now.